Jenkinsfile

```diff
@@ -199,17 +189,20 @@ node('ca-jenkins-agent') {
         header: "## Recent Changes"
     )
 
-    // Deploys the application if on a protected branch. Give the version input
-    // 30 minutes before an auto timeout approve.
-    pipeline.deploy(
-        versionArguments: [timeout: [time: 30, unit: 'MINUTES']]
+    // Perform the versioning email mechanism
+    pipeline.version(
+        timeout: [time: 30, unit: 'MINUTES']
```

Comment:
Thanks for splitting! 👍 
Maybe now we can cut a 2.0.0 of the shared libraries (version-controller) repo

Reply:
That'd be nice. 😄